tests/conftest.py

```diff
@@ -1673,15 +1674,23 @@ async def _async_call(
         domain: str,
         service: str,
         service_data: dict[str, Any] | None = None,
-        **kwargs: Any,
+        blocking: bool = False,
+        context: Context | None = None,
+        target: dict[str, Any] | None = None,
+        return_response: bool = False,
```

Comment:
One advantage of `**kwargs` was that we didn't need to update the fixture every time the signature of `async_call` was slightly modified / a new argument added.

I agree there is value in having `context` here but `target` and `return_response` could still be behind `**kwargs`, can't they?

Reply:
That was my original idea also, when I first implemented it with just three arguments.
Now, since `return_response` also gets passed to `ServiceCall` I think it's better to simply align it fully.